packages/support/src/SupportServiceProvider.php

```diff
@@ -67,7 +67,8 @@ public function packageBooted()
                     ->filter(fn (string $package): bool => InstalledVersions::isInstalled("filament/{$package}"))
                     ->join(', '),
                 'Views' => function () use ($packages): string {
-                    $publishedViewPaths = collect($packages)->filter(fn (string $package): bool => is_dir(resource_path("views/vendor/{$package}")));
+                    $publishedViewPaths = collect($packages)
```

Comment:
@danharrin you could use `array_filter` here.

Reply:
Nah, we're using `count()` and `join()` after

Reply:
Should be able to implode.